### DISCUSSION TITLE
Will i3 allow me to create an empty named container

### DISCUSSION THREAD

**Discussion by boufe16** [i3/i3#5997] [Q&A] [answered]
Hello! Say that when I start a group of apps, I'd like to move 3 or 4 of them as a unit. lets say its an empty container named "chat".
can i create this chat container, assign my windows to "chat" and use for_window commands in my config on this "chat" group I created? 

The alternative would be to create a script that does all my move and tabbing commands in sequence I presume? then if I want to move on the fly I would $Mod+a to select the group and do whatever? 

Thanks :)

**Comment by orestisfl** [comment, marked as answer]
You might be interested in [Layout saving in i3](https://i3wm.org/docs/layout-saving.html)

**Reply by boufe16** [reply]
Nice! I'll try this out, looks exactly like what I want. Thanks!
